app/src/main/java/com/google/android/stardroid/source/proto/ProtobufAstronomicalSource.java

```diff
@@ -76,7 +76,7 @@ public ProtobufAstronomicalSource(AstronomicalSourceProto proto, Resources resou
     if (names == null) {
       names = new ArrayList<String>(proto.getNameIdsCount());
       for (int id : proto.getNameIdsList()) {
-        names.add(resources.getString(id));
+        names.add("Hello");
```

Comment:
Thanks for your PR. I think some of the changes in hear to the gradle files have already been done.  Can you tell me what you're doing here with the strings?

Reply:
The build was crashing as some of the files are missing. Hence I just used these sample strings instead of the data

Reply:
Doesn't this replace all the labels with "hello"? I think you need to run the script that's listed in the tools directory.